xbmc/interfaces/json-rpc/InputOperations.cpp

```diff
@@ -84,6 +88,45 @@ JSONRPC_STATUS CInputOperations::ExecuteAction(const std::string &method, ITrans
   return SendAction(action);
 }
 
+JSONRPC_STATUS CInputOperations::Button(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  std::string button = parameterObject["button"].asString();
+  std::string keymap = parameterObject["keymap"].asString();
+
+  int keycode = 0;
+  if ( keymap.compare("KB") == 0 ) // standard keyboard map
+  {
+  keycode = CKeyboardTranslator::TranslateString( button.c_str() );
+  }
+  else if  ( keymap.compare("XG") == 0 ) // xbox gamepad map
+  {
+  keycode = CGamepadTranslator::TranslateString( button.c_str() );
```

Comment:
> Use spaces as tab policy with an indentation size of 2.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#32-indentation

`CKeyboardTranslator::TranslateString` takes a `std::string` as argument, so there is no need to call `.c_str()`.